stable/rocketchat/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if gt .Values.replicaCount 1.0 }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app: {{ template "rocketchat.name" . }}
```

Comment:
Please use the "new" style labels (i.e. `app.kubernetes.io/name`)

Reply:
Ah, thanks - fixed